src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -263,6 +271,37 @@ def _create_batch_script(job_dir, task_list_file, n_jobs, tool):
     return batch_script_file
 
 
+def _create_job_dir(root_dir, tool, kbase_collection, load_ver, force=False):
```

Comment:
So right now the script doesn't check the job dir to see what's already run, right, and it's just storing what was just run? That's coming later?

Reply:
:+1: